x-pack/plugins/apm/server/lib/search_strategies/correlations/utils.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export function* range(start: number, end: number, step: number) {
+  while (start < end) {
+    yield start;
+    start += step;
+  }
+}
+
+// https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/random
```

Comment:
This attribution should be expanded following the MDN [recommendation](https://developer.mozilla.org/en-US/docs/MDN/About#copyrights_and_licenses). Suggest something like

```
From https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/random by Mozilla Contributors licensed under CC-BY-SA 2.5
```

Reply:
Fixed here https://github.com/elastic/kibana/pull/99905/commits/0331fc1c3fcdc1ac3a8344e8839d4ca56ec484de